Jenkinsfile

```diff
@@ -14,19 +14,19 @@ pipeline {
         }
     }
     stages {
-        stage('Test and Deploy Extensions') {
+        stage('Build and Test') {
             steps {
                 checkout scm
                 dir('spring-filters-config') {
-                    git branch: 'fixMeteringIntegTests', changelog: false, credentialsId: 'github.build.ge.com', poll: false, url: 'https://github.build.ge.com/predix/spring-filters-config.git'
+                    git branch: 'master', changelog: false, credentialsId: 'github.build.ge.com', poll: false, url: 'https://github.build.ge.com/predix/spring-filters-config.git'
```

Comment:
checked spring-filters-config, set-env-metering-filter.sh is different in fixMeteringIntegTests vs. master. not sure why switch to master.

Reply:
@anny1099, I merged changes to set-env-metering-filter.sh into master.  There is no reason to keep these changes on a branch.